pubspec.yaml

```diff
@@ -18,7 +18,8 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 0.0.1+1
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '2.13.0'
```

Comment:
We can be more loose with Dart version, ">=2.12.0" is probably fine. Dart is packaged with Flutter so by pinning Flutter version, we are also pinning Dart version. Lower Dart constraint is just to tell compiler we are using sound null-safety.